src/clients/mockApi.ts

```diff
@@ -206,8 +206,15 @@ export async function changeStatus({
     },
   ])
 
+  const nextAggregates = {
```

Comment:
If we're going to mutate here let's just use lodash's `cloneDeep` which does the same thing and requires less effort to scan.

Otherwise, this is a functional approach that I think is clear

```typescript
_.merge({}, issue.aggregates, {
  comments: {
    count: issue.aggregates.count + 1
  }
})
```